### PR TITLE
Leonard/multi modal

### PR DESCRIPTION
**Pull request by leonard-q**
:star2: Hello! Thanks for contributing JORLDY! 

### Checklist 

Please check if you consider the following items. 

- [v] My code follows the style guidelines of this project
- [v] My code follows the [naming convention](https://github.com/kakaoenterprise/JORLDY/blob/master/docs/Naming_convention.md) of documentation
- [v] I have commented my code, particularly in hard-to-understand areas
- [v] My changes generate no new warnings or errors 

### Types of changes 

**Please describe the types of changes! (ex. Bugfix, New feature, Documentation, ...)**
New feature

### Test Configuration

- OS: Linux Ubuntu
- Python version: 3.8
- Additional libraries: None 

### Description

**Please describe the details of your contribution**
Envs which have Multi modal (image, vector) input can be applied to all agents.

### CODE REVIEW

**Comment by leonard-q** [comment]
The multi modal is tested without an error in the following environment
image

File: jorldy/core/buffer/base.py [hunk 1]
```diff
@@ -33,3 +38,18 @@ def sample(self, batch_size):
...
         
+    def stack_transition(self, batch):
+        transitions = {}
+
+        for key in batch[0].keys():
+            if len(batch[0][key]) > 1:
+                # Multimodal
+                b_list = []
+                for i in range(len(batch[0][key])):
+                    temp_transition = np.stack([b[key][i][0] for b in batch], axis=0)
```
Comment (ramanuzan) [inline jorldy/core/buffer/base.py:49]: temp -> tmp

File: jorldy/core/env/ml_agent.py [hunk 2]
```diff
@@ -64,14 +64,39 @@ def step(self, action):
...
+        next_state = self.state_config(term.obs) if done else self.state_config(dec.obs)
         
         self.score += reward[0] 
 
         reward, done = map(lambda x: np.expand_dims(x,0), [reward, [done]])
-        
+
         return (next_state, reward, done)
 
+    def state_config(self, obs):
```
Comment (ramanuzan) [inline jorldy/core/env/ml_agent.py:75]: state_config -> state_processing

일반화 어려우면 부모 클래스에  obs -> obs[0] 를 default로 만들고 자식 환경에서 specific하게 구현하도록 하는 방법으로 진행해도 좋을 것 같습니다.

**Comment by leonard-q** [comment]
Codes are modified based on the comment! :) @ramanuzan

**Pull request merged**